src/rpc/core_rpc_server.cpp

```diff
@@ -583,6 +583,46 @@ namespace cryptonote
 
     CHECK_PAYMENT(req, res, 1);
 
+    res.daemon_time = time(NULL);
```

Comment:
Keeping this here in case you need it.
```cpp
#include <chrono>

std::chrono::time_point<std::chrono::system_clock> now = std::chrono::system_clock::now();
auto duration = now.time_since_epoch();
// 64bit; microseconds since epoch
int64_t daemon_time = std::chrono::duration_cast<std::chrono::microseconds>(duration).count();
```

Reply:
I was indeed thinking about switching to more "modern" time and clock functions but then saw that the data types in the RPC interface all simply use `time_t`, and I wasn't even sure whether the binary and JSON serialization code that Monero uses could handle those "chrono" types. So I decided to stick with good old `time_t` and also directly use the `time` function.

Reply:
Quotations is definitely the way to go when it takes 3 long lines to do something Python can do in one without complications 😂.

Reply:
And 5 minutes after writing the above I learned that `time_t` is **also** problematic in the RPC interface and **not** previously used already as I believed. The way to go at least for the interface type definitions is `uint64_t` ...